src/js/views/globals/app-nav/app-nav.scss

```diff
@@ -4,6 +4,12 @@ $menu-hover-color: #375467;
   color: $black-90;
   height: 100%;
   position: relative;
+  transition: width 250ms ease-in-out 0s;
+  width: 200px;
```

Comment:
Simple transition of `250s` whenever the width of nav is changed. This is what Shortcut uses for their nav menu, so copied it from them. Can change this to something else if we want to.